docs/anchoring-assets/anchor-platform/events/integration.mdx

```diff
@@ -0,0 +1,140 @@
+
+## Receiving Events
+
+The event service can be used to send events to client and business server callback endpoints. The event service will send events to these endpoints as HTTP POST requests with the event data in the request body.
+
+### As a Client Application
+
+Client applications can receive updates whenever the `status` of a SEP transaction changes.
+
+To receive events as a client application, you will need to expose a callback URL that the event service can send events to. The event service will send a POST request to this endpoint with the event data in the request body.
+
+Anchor Platform will only send events to clients listed in the client configuration. See the client configuration documentation for more information.
+
+#### Callback Signing
+
+Anchor Platform signs the callback requests it sends to client applications. The signature is included in the `Signature` header of the request. The callback URL signature specification can be found in the corresponding SEP protocol specifications.
+
+- [SEP-0006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#url-callback-signature)
+- [SEP-0024](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#url-callback-signature)
+- [SEP-0031](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#url-callback-signature)
+
+### As a Business Server
+
+In addition to SEP transaction status updates, business servers can receive events about SEP-31 quote creation or SEP-12 customer information updates. The schema of the event data will depend on the type of event being sent. Visit the [Event API documentation](/api/anchor-platform/callbacks/post-event) for more information about the schema of the event data.
+
+To receive events as a business server, you will need to expose a callback URL that the event service can send events to. The event service will send a POST request to this endpoint with the event data in the request body.
+
+#### Configuration
+
+The event service's callback API can be configured using the `callback_api` section of the Anchor Platform configuration file. The following is an example of how to configure the event service's callback API with JWT authentication:
+
+<CodeExample>
+
+```yaml
+callback_api:
+  base_url: http://localhost:8081/callback
+  secret: my-secret
+  auth:
+  type: jwt
+  jwt:
+    expiration_milliseconds: 30000
+    http_header: Authorization
+```
+
+</CodeExample>
+
+The following is an example of how to configure the event service's callback API with API key authentication:
+
+<CodeExample>
+
+```yaml
+callback_api:
+  base_url: http://localhost:8081/callback
+  secret: bf7c2c5b-5c76-4974-a318-ac9c4a2b8514
+  auth:
+  type: api_key
+  jwt:
+    http_header: X-Api-Key
+```
+
+:::caution
+
+The `http_header` is configurable as of version `2.5.2` and earlier. The default value is `Authorization` for JWT and `X-Api-Key` for API key.
```

Comment:
Should it say that it's configurable, but not working as intended? (Judging by the bug description)

Reply:
Oops, I meant _not_ configurable. Thanks for catching this.